libs/deal-ticket/src/hooks/use-type-store.ts

```diff
@@ -0,0 +1,28 @@
+import { create } from 'zustand';
+import { persist, subscribeWithSelector } from 'zustand/middleware';
+
+export enum DealTicketType {
+  Limit = 'Limit',
+  Market = 'Market',
+  StopLimit = 'StopLimit',
+  StopMarket = 'StopMarket',
+}
+
+export const useDealTicketTypeStore = create<{
```

Comment:
IMO this should just be part of the main deal ticket store, we shouldn't need a separate store just for the type

Reply:
I agree, it will be refactored in follow up PR